src/locales/en.json

```
@@ -100,6 +100,9 @@
   "footer_nav_help": "Help",
   "footer_nav_doc": "Doc",
   "footer_nav_faq": "FAQ",
+  "order_menu_price":"By Price",
+  "order_menu_time":"By Time",
+  "order_menu_collected":"By Own",
```

Can use [interpolation](https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) here
```suggestion
  "order_menu_price":"Price",
  "order_menu_time":"Time",
  "order_menu_collected":"Own",
  "order_menu_by":"By {order}",
```